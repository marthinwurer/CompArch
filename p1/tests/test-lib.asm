; test the lib instruction
	.entry	start
;
	.org	0x200
start:
	lib	1
;
	lib	5
;
	lib	0x7ff
;
	lib	0xfff
;
	lib	0xffb
;
	lib	0x800
;
	halt
