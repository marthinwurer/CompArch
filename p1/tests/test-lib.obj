200 1 17001
201 1 17005
202 1 177ff
203 1 17fff
204 1 17ffb
205 1 17800
206 1 00000
200
