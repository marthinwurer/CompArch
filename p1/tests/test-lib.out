CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 200
200:  17001  LIB   001  A[00000]  B[00000]  XR[000]
201:  17005  LIB   005  A[00000]  B[00001]  XR[000]
202:  177ff  LIB   7ff  A[00000]  B[00005]  XR[000]
203:  17fff  LIB   fff  A[00000]  B[007ff]  XR[000]
204:  17ffb  LIB   ffb  A[00000]  B[fffff]  XR[000]
205:  17800  LIB   800  A[00000]  B[ffffb]  XR[000]
206:  00000  HALT       A[00000]  B[ff800]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 22 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
